docs/README.md

```diff
@@ -163,6 +163,19 @@ The most important root project Gradle tasks are listed in the table below.
 | funTest     | Run functional tests for all projects                             |
 | installDist | Build all projects and install the start scripts for distribution |
 
+### Extending ORT
+
+ORT provides extension points which can be used by plugins to extends the functionality of ORT. For details about the
+extension points see the [readme of the example plugin project](../examples/plugin/README.md).
+
+In general it is preferred that extensions to ORT are contributed directly to this repository. But there are exceptions
```

Comment:
I believe we should be more consistent about the terms "extension" and "plugin" here. I.e. all bullet items below should talk about "extension(s)", and maybe then add a sentence after the last bullet item saying "In such cases it is possible to implement the extension as a separate plugin".